crates/cli/commands/src/node.rs

```diff
@@ -110,6 +110,10 @@ pub struct NodeCommand<
     /// Additional cli arguments
     #[command(flatten, next_help_heading = "Extension")]
     pub ext: Ext,
+
+    /// Engine cli arguments
+    #[command(flatten, next_help_heading = "Extension")]
```

Comment:
```suggestion
    #[command(flatten, next_help_heading = "Engine")]
```

and move above `ext`